contributing/developer-blog.md

```diff
@@ -0,0 +1,66 @@
+
+### What is the process once my blog is accepted?
+
+Once a blog is accepted, we’ll ask you for a date when we can expect the draft by. Typically we’ll ask that you can commit to having this ready within a month of submitting the issue.
+
+Once you submit a draft, we’ll return a first set of edits within 5 business days.
+
+The typical turnaround time from issue creation to going live on the developer blog is ~4 to 6 weeks.
+
+### What happens after my blog is published?
+
+We’ll share the blog on the dbt Labs social media channels! We also encourage you to share on the dbt Slack in #i-made-this.
+
+### What if my post doesn’t get approved?
+
+We want to publish as many community contributors as possible, but not every post will be a fit for the Developer Blog. That’s ok! There are many different reasons why we might not be able to publish a post right now and none of them reflect on the quality of the proposed post.
+
+- **dbt Labs capacity**: We’re committed to providing hands-on feedback and coaching throughout the process. Our goal is not just to generate great developer blogs - it’s to help build a community of great writers / practitioners who can share their knowledge with the community for years to come. This necessarily means we will be able to take on a lower absolute number of posts in the short term, but will hopefully be helpful for the community long term.
+- **Posts Should Focus on Narrative / Problem Solving - Not Industry Trends**: The developer blog exists, primarily, to tell the stories of analytics engineering practitioners and how they solve problems. The idea is that reading the developer blog gives a feel for what it is like to be a data practitioner on the ground today. This is not a hard and fast rule, but a good way to approach this is “How I/we solved X problem” rather than “How everyone should solve X problem”.
+
+We are very interested in stacks, new tools and integrations and will happily publish posts about this - with the caveat that the *focus* of the post should be solving real world problems. Hopefully if you are writing about these, this is something that you have used yourself in a hands on, production implementation.
+
+- **Posts Should Have Right Sized Scope**: We want to be able to cover a topic in-depth and dig into the nuances. Big topics like “How should you structure your data team” or “How to ensure data quality in your organization” will be tough to cover in the scope of a single post. If you have a big idea - try subdividing it! “How should you structure your data team” could become “How we successfully partnered with our RevOps team on improving lead tracking” and “How to ensure data quality in your organization” might be “How we cleaned up our utm tracking”.
```

Comment:
I think of one of our (potentially undocumented) stylistic pushes is 'DRY Lists' -- 'Post should' twice indicates maybe that part could just be dropped?

Reply:
✔️